### ISSUE TITLE
[Accessibility], WCAG 4.1.2. search field’s Clear button (X) has no accessible name.

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1974]
## URL address of the page where you encountered the problem
https://test.dev.finto.fi/en/

## Description of the problem
The Clear -button (X) has no accessible name. When screen reader focus moves to the button, screen reader is unable to tell the name of the button, leaving its purpose completely unclear.

**Fix**: Add an accessible name, e.g. aria-label="Clear search field". Make sure the accessible name is in the language of the main language of the site.

**How to test:** Turn on NVDA screen reader. Type something to the search field and then press tab key:

**Expected result:** Screen reader focus moves to X-button and screen reader says “Clear search field button”.

**Actual result:** Screen reader focus moves to X-button and screen reader says “button”.

Severity: Serious
Priority: High

## Additional information (e.g. screenshots) about the problem

<img width="908" height="341" alt="Image" src="https://github.com/user-attachments/assets/1b399406-9c72-4033-81ac-825578c8ae40" />

## The browser you used when the problem appeared
Chrome, Firefox, Edge

**Comment by tinskuu** [comment]
This issue is fixed.
